src/hocs/withCheckNetworkConnection/withCheckNetworkConnection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Alert } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { selectors } from 'app/state/electrumX';
+
+type checkNetworkConnectionArgs = (...args: any[]) => void;
```

Comment:
it looks like you use it just like a callback thus I'd name it like this. Also, I think types should start with a capital letter (https://basarat.gitbook.io/typescript/styleguide#type).

Reply:
changed